.github/workflows/ci.yml

```
@@ -21,20 +23,24 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v4
+    - name: CMake Configure
+      run: cmake -B build -S . -DCMAKE_BUILD_TYPE=Debug -DSERIAL=1
```

`-DSERIAL` no longer used..

```suggestion
      run: cmake -B build -S . -DCMAKE_BUILD_TYPE=Debug
```